apps/transport/lib/transport_web/templates/page/real_time.html.md

```diff
@@ -2,12 +2,13 @@
 
 Le Point d'Accès National aux données de transport (PAN) fournit un accès **homogène** à ces données pour en faciliter l’intégration et la réutilisation.
 
-Les exigences pour publication homogène de données temps-réel de lignes régulières (bus etc.) sont détaillées [sur cette page](https://doc.transport.data.gouv.fr/producteurs/operateurs-de-transport-regulier-de-personnes/temps-reel-des-transports-en-commun). 
+Les exigences pour publication homogène de données temps-réel de lignes régulières (bus etc.) sont détaillées [sur cette page](https://doc.transport.data.gouv.fr/producteurs/operateurs-de-transport-regulier-de-personnes/temps-reel-des-transports-en-commun).
 
 Cependant, certains réseaux de transport disposent de données temps-réel non standardisées ou ne remplissants pas les conditions de licences de la plateforme. Celles-ci n'ont pas vocation à être référencées sur le PAN en l'état, mais elles sont listées ci-dessous pour référence.
 
 
 <table class="table">
+<tr>
```

Comment:
oh, you found them, nice!

Reply:
hell  yeah!
the `<tr>` missing wasn’t the problem (line 11)

The `<table>` was closing, so I struggled to find it… it’s the one missing `</td>` that yielded the 3 warnings…